src/main/java/com/odysseusinc/arachne/executionengine/service/impl/RuntimeServiceImpl.java

```diff
@@ -172,6 +175,9 @@ public void analyze(AnalysisRequestDTO analysis, File file, ResultCallback resul
                     } finally {
                         if (!isExternalJail()) {
                             FileUtils.deleteQuietly(runFile);
+                            for (Path path : tmpPaths) {
```

Comment:
I see the tmpPaths is used only to cleanup temporary files, but why not do that outside of this method?
Moreover, it looks like some kind of anti-patterns.